extension-spring/pom.xml

```diff
@@ -56,7 +56,18 @@
             <version>2.5</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>javax.validation</groupId>
+            <artifactId>validation-api</artifactId>
+            <version>2.0.0.Final</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Please update this to use jakarta.validation:jakarta.validation-api:2.0.2

Reply:
Sure. I was pulling from other projects.. not an issue.